force-app/main/default/lwc/assetManagementRelated/assetManagementRelated.js

```diff
@@ -0,0 +1,83 @@
+import { LightningElement, wire, track, api} from 'lwc';
+import getAssets from '@salesforce/apex/AssetManagementController.getAssetsByAccount'
+import renewCancel from '@salesforce/apex/AssetManagementController.renewCancelAsset'
+const columns = [
+    { label: 'Asset Name', fieldName: 'Name', type: 'text' },
+    { label: 'Lifecycle Start Date', fieldName: 'LifecycleStartDate',type: 'date', typeAttributes: {
```

Comment:
What happens when users don't have access to one or more of these fields, how does this component handle that?

Reply:
Columns will be visible and the data will not show up for that particular column